test_tracetools/CMakeLists.txt

```diff
@@ -36,13 +36,36 @@ if(BUILD_TESTING)
   find_package(lifecycle_msgs REQUIRED)
   find_package(rclcpp REQUIRED)
   find_package(rclcpp_lifecycle REQUIRED)
+  find_package(rcpputils REQUIRED)
   find_package(std_msgs REQUIRED)
   find_package(std_srvs REQUIRED)
 
+  # The utility lib is needed even if TRACETOOLS_DISABLED; it's just empty
+  add_library(${PROJECT_NAME} src/mark_process.cpp)
```

Comment:
This is minor, but you might want to call this library `mark_process` or something like that.  In other words, `libtest_tracetools.so` doesn't tell me much about what it does (and why a test package is installing it), but `libmark_process.so` is a little more descriptive.

But this is only a suggestion, feel free to ignore it if you prefer to keep it as-is.

Reply:
Yeah, that's fair. I renamed it to `test_tracetools_mark_process`: https://github.com/ros2/ros2_tracing/compare/cdab11e618ce64a089bdd391699da96894536d55..edeaa9454d15e05865b08d625f15e2c7ecbb8dc7.